apps/demo/src/app/app.module.ts

```diff
@@ -37,8 +40,10 @@ import { DemoDriverMap } from './drivers/map';
     BrowserAnimationsModule,
 
     HttpClientModule,
-
-    DemoDriverMap[environment.driver.variant],
```

Comment:
This caused a "Value at position X in the NgModule.imports of AppModule is not a reference" error. I'm not sure what the best way to do this is right now, but since I plan on pushing through to Angular 10, I think we just call this tech debt and move forward. The way I have it does work